Description: Work around gcc bug for successful ARMv7 build
Author: lordpipe <lordpipe@protonmail.com>
Bug: https://github.com/PrismLauncher/PrismLauncher/issues/128
Reviewed-by: Ben Westover <me@benthetechguy.net>
Last-Update: 2023-07-30

--- a/libraries/libnbtplusplus/include/tag_primitive.h
+++ b/libraries/libnbtplusplus/include/tag_primitive.h
@@ -77,14 +77,6 @@
 typedef tag_primitive<float> tag_float;
 typedef tag_primitive<double> tag_double;
 
-//Explicit instantiations
-template class NBT_EXPORT tag_primitive<int8_t>;
-template class NBT_EXPORT tag_primitive<int16_t>;
-template class NBT_EXPORT tag_primitive<int32_t>;
-template class NBT_EXPORT tag_primitive<int64_t>;
-template class NBT_EXPORT tag_primitive<float>;
-template class NBT_EXPORT tag_primitive<double>;
-
 template<class T>
 void tag_primitive<T>::read_payload(io::stream_reader& reader)
 {
